.env

```diff
@@ -16,3 +16,5 @@ IR_IMAGE=nspccdev/neofs-ir
 # NeoFS Storage nodes
 NODE_VERSION=0.12.0-rc1
 NODE_IMAGE=nspccdev/neofs-storage
+
+NEOFS_CONTRACT_MAINCHAIN=af5dc5f7e6a6efc64d679098f328027591a2e518
```

Comment:
It's already in `services/ir/.ir.env`. Common .env should not have volatile serivice-specific settings.

Reply:
yeah, but it contains IR specific environments, that should not be accessible by a another services, isn't it?